public/test/utils/regions/story.js

```diff
@@ -13,10 +16,28 @@ class Story {
         return group(1, this.dom, this);
     }
 
-    pacakgeSelector() {
-        return this.dom.querySelector('.front-selector');
+    getPackageInSelector(id) {
+        return $('.front-selector option[value=' + id + ']');
     }
-}
+
+    getPackageInSelectorText(id) {
+        return textInside('.front-selector option[value=\"' + id + '\"]');
```

Comment:
to avoid duplication and possible errors, I'd use `return textInside(this.getPackageInSelector(id))`